### DISCUSSION TITLE
@hookstate/devtools - unable to resolve dependency tree

### DISCUSSION THREAD

**Discussion by d3v3lope** [avkonst/react-use-state-x#400] [General]
Hi. I am trying to install `npm i @hookstate/devtools`. Unfortunately I get an error (screenshot below). Are you planning to implement `@hookstate/devtools` in version 4.1 with bumped dependencies?  
![image](https://github.com/avkonst/hookstate/assets/85257850/d4e56566-5536-4f0d-84c1-9cd63fdac65b)

**Comment by avkonst** [comment]
fixed in devtool 4.0.1.. it was a bug in devtools package.json peer dependencies.
